src/ui/views/Gallery/gallery.ts

```diff
@@ -33,6 +30,9 @@ export function getCoverRealPath(
 
 function getResourcePathFromLinkText(app: App, text: string) {
   const linkText = parseObsidianLink(text)?.linkText || text;
+  if (linkText.startsWith("http://") || linkText.startsWith("https://")) {
+    return linkText;
+  }
```

Comment:
A plain web link will get `null` return from `parseObsidianLink()`, then fall back to the original text.
Both wrapped external link and the plain web link got recognized and returned "as is" from these lines.